### DISCUSSION TITLE
unable to downgrade my github plan

### DISCUSSION THREAD

**Discussion by ankur1163** [community/community#46103] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i have tried 2-3 times to downgrade the package, but page reloads and i am still on paid plan, can someone help?

**Comment by DanteHaxor** [comment]
You can always mail them about the same

**Comment by jge162** [comment]
I have seen this issue multiple times. you need to submit a billing [Github support ticket](https://support.github.com/contact/account?tags=hubberfy_account)
